spec/controllers/concerns/locale_detection_sprec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
```

Comment:
# frozen_string_literal: true 

is missing ;) will be obsolete with ruby 3.4.0 but for now we need it

Reply:
current lived standard is without (at least in this repository, for the specs) :-P

Reply:
😄 👍

Reply:
I mean it's obsolete soon